src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -213,8 +212,20 @@ protected final String buildMethodName() {
     }
 
     protected final AnnotationSpec generatedAnnotation() {
+        Class<?> generatedAnnotation;
```

Comment:
I'm not a huge fan of this variable name, mainly because it's the same as the method.

How about `generatedAnnotationClass` instead?

Reply:
Done with separate helper method below.